sklearnex/linear_model/coordinate_descent.py

```diff
@@ -17,10 +17,14 @@
 from daal4py.sklearn.linear_model import ElasticNet, Lasso
 from onedal._device_offload import support_usm_ndarray
 
+# Note: `sklearnex.linear_model.ElasticNet` only has functional
+# sycl GPU support. No GPU device will be offload.
 ElasticNet.fit = support_usm_ndarray(queue_param=False)(ElasticNet.fit)
 ElasticNet.predict = support_usm_ndarray(queue_param=False)(ElasticNet.predict)
 ElasticNet.score = support_usm_ndarray(queue_param=False)(ElasticNet.score)
 
+# Note: `sklearnex.linear_model.Lasso` only has functional
+# sycl GPU support. No GPU device will be offload.

```

Comment:
is it a concern that we will be removing sycl GPU interfaces?

Reply:
Regarding Lasso, Ridge, and ElasticNet, it’s worth noting that there is no GPU offloading for these specific algorithms.
However, with dpct/dpnp and a GPU SYCL queue, you can still use them as input and output, but all algorithm computations will occur on the CPU device, even when a GPU SYCL queue is provided.